lib/vrl/stdlib/src/parse_cef.rs

```diff
@@ -129,6 +129,13 @@ fn parse(input: &str) -> Result<impl Iterator<Item = (String, Value)> + '_> {
         let result = extension
             .into_iter()
             .chain(headers)
+            .map(|(key, mut value)| {
+                // Strip quotes from value
+                if value.starts_with('"') && value.ends_with('"') {
+                    value = value[1..value.len() - 1].to_string();
```

Comment:
edge case here: If the value is just a single quotation mark, this will panic. Maybe check that the length is at least 2?